build/Common.props

```diff
@@ -7,7 +7,6 @@
     <DefineConstants>$(DefineConstants);SIGNED</DefineConstants>
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
     <NetFrameworkMinimumSupportedVersion>net462</NetFrameworkMinimumSupportedVersion>
-    <NetMinimumSupportedVersion>net6.0</NetMinimumSupportedVersion>
```

Comment:
I would change it to .net8 and replace in all "production" csproj by the new value - separate PR when all other are merged.

Reply:
Yes, when all others will be merged, I'll follow with few cleanup PRs. Thank you!